stable/consul/README.md

```diff
@@ -48,13 +47,15 @@ The following tables lists the configurable parameters of the consul chart and t
 | `ui.enabled`            | Enable Consul Web UI                  | `false`                                                    |
 | `uiService.enabled`      | Create dedicated Consul Web UI svc    | `false`                                                    |
 | `uiService.type`        | Dedicate Consul Web UI svc type       | `NodePort`                                                 |
+| `test.image`        | Test container image requires kubectl + bash (used for helm test)      | `lachlanevenson/k8s-kubectl`                                                 |
```

Comment:
Yes, I think this is fine. I've documented the prerequisites to cut your own container to run the tests if desired